build.gradle

```diff
@@ -15,9 +29,21 @@ buildscript {
 
 allprojects {
     repositories {
+        mavenCentral()
         google()
-        jcenter()
-        maven { url "https://raw.githubusercontent.com/guardianproject/gpmaven/master" }
         maven { url 'https://jitpack.io' }
+        maven {
+            url "https://raw.githubusercontent.com/guardianproject/gpmaven/master"
+            content {
+                includeGroup('info.pluggabletransports.aptds')
```

Comment:
we no longer need this since we are using iptproxy from jitpack

Reply:
There is still this from that repo:

```gradle
implementation 'info.pluggabletransports.aptds:jsocksAndroid:1.0.4'
```